yjit/src/backend/x86_64/mod.rs

```diff
@@ -346,9 +364,7 @@ impl Assembler
                     add(cb, insn.opnds[0].into(), insn.opnds[1].into());
                 },
 
-                Op::Breakpoint => int3(cb),
-
-                _ => panic!("unsupported instruction passed to x86 backend: {:?}", insn.op)
```

Comment:
I think we do want to keep the `_ => panic!(...)` because some instructions that we feed to the backend could get lowered into other instructions. So it's possible that some of our backend instructions can never make it to the emit stage.

Reply:
We could also add a comment explaining what I just said above.